decorator.rb

```diff
@@ -0,0 +1,14 @@
+require './nameable'
+
+class Decorator < Nameable
+  attr_accessor :nameable
+
+  def initialize()
+    super()
+    @nameable = nameable
```

Comment:
- [ ] [OPTIONALY] Currently your decoration class is not working

![Screenshot 2023-08-29 150719](https://github.com/aima98/OOP-School-Library1/assets/57408419/be643669-e698-44c2-8a55-af495ba5b065)

Kindly make sure you initialize the   `nameable`, it will be something like this:

```ruby
    .
    def initialize(nameable)
      @nameable = nameable
    end
    .
```

after changing that line, I got the result 

![Screenshot 2023-08-29 151442](https://github.com/aima98/OOP-School-Library1/assets/57408419/89cca1bb-ff0b-45aa-8250-94e9303c4e35)